### PR TITLE
Updated Docs related to View component

### PR DESCRIPTION
**Pull request by stevenyuen**

### CODE REVIEW

File: demo/content/View/index.js [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+            <Spacer size="9" />
+            <Grid columns="1[a] 2[b] 4[c-e]" gutter="60px">
+               <GridCell>
+                  <Title>Introduction</Title>
+               </GridCell>
+               <GridCell colspan="3">
+                  <Text size="6" weight="light">
+                     The View component is probably one of the most important components within OIO.
+                     It is the core OIO component you should use to structure your pages, layouts and components for your application.
+                     You can think of the <code>View</code> component as OIO‘s take on a <code>div</code>.
```
Comment (asabhaney) [inline demo/content/View/index.js]: Sorry if this is a dumb question, but is that apostrophe intended to be like that? It looks like a reverse apostrophe haha.

Comment (stevenyuen) [inline demo/content/View/index.js]: hmm yeah I noticed that. I think that's what happens when I use that 'safe' apostrophe (Option-])

**Comment by stevenyuen** [comment]
@asabhaney made some updates. Tossing back to you.

**Comment by asabhaney** [comment]
Nice work

**Pull request merged**